docs/source/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,730 @@
+      "Given label: O, predicted label: ORG\n",
+      "\n",
+      "Sentence 9430, token 10: \n",
+      "The revered Roman Catholic nun was admitted to the Calcutta \u001b[31mhospital\u001b[0m a week ago with high fever and severe vomiting.\n",
+      "Given label: LOC, predicted label: O\n",
+      "\n",
+      "Sentence 6143, token 25: \n",
+      "The embattled Afghan government said last week that the Kabul-Salang highway would be opened on Monday or Tuesday following talks with the Supreme Coordination Council \u001b[31malliance\u001b[0m led by Jumbish-i-Milli movement of powerful opposition warlord General Abdul Rashid Dostum.\n",
+      "Given label: ORG, predicted label: O\n",
+      "\n",
+      "Sentence 18367, token 0: \n",
+      "\u001b[31mCan\u001b[0m/ U.s. Dollar Exchange Rate: 1.3570\n",
+      "Given label: LOC, predicted label: O\n",
+      "\n",
+      "Sentence 12049, token 0: \n",
+      "\u001b[31mBorn\u001b[0m in 1937 in the central province of Anhui, Dai came to Shanghai as a student and remained in the city as a prolific author and teacher of Chinese.\n",
+      "Given label: LOC, predicted label: O\n",
+      "\n",
+      "Sentence 16764, token 7: \n",
+      "1990 - British historian Alan John Percivale \u001b[31m(\u001b[0mA.j.p.) Taylor died.\n",
+      "Given label: PER, predicted label: O\n",
+      "\n",
+      "Sentence 20446, token 0: \n",
+      "\u001b[31mPace\u001b[0m bowler Ian Harvey claimed three for 81 for Victoria.\n",
+      "Given label: PER, predicted label: O\n",
+      "\n",
+      "Sentence 15514, token 16: \n",
+      "But one must not forget that the Osce only has limited powers there,\" said \u001b[31mCotti\u001b[0m, who is also the Swiss foreign minister.\"\n",
+      "Given label: O, predicted label: PER\n",
+      "\n",
+      "Sentence 7525, token 12: \n",
+      "Specter met Crown Prince Abdullah and Minister of Defence and Aviation Prince \u001b[31mSultan\u001b[0m in Jeddah, Saudi state television and the official Saudi Press Agency reported.\n",
+      "Given label: PER, predicted label: O\n",
+      "\n",
+      "Sentence 2288, token 0: \n",
+      "\u001b[31mSporting\u001b[0m his customary bright green outfit, the U.s. champion clocked 10.03 seconds despite damp conditions to take the scalp of Canada's reigning Olympic champion Donovan Bailey, 1992 champion Linford Christie of Britain and American 1984 and 1988 champion Carl Lewis.\n",
+      "Given label: ORG, predicted label: O\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "display_issues(issues, given_words, pred_probs=pred_probs, given_labels=labels, \n",
+    "               exclude=[(0, 1), (1, 0)], class_names=merged_entities) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96d04902",
+   "metadata": {},
+   "source": [
+    "More than half of the potential label issues are identified correctly. As shown above, some examples are ambigious and require manual checking. "
```

Comment:
"are identified correctly."
-> 
"correspond to tokens that are incorrectly labeled."